docs/evaluate/temporal-cloud/overview.mdx

```diff
@@ -0,0 +1,77 @@
+Your Workers and Temporal Application connect directly to this self-hosted instance.
+With the self-hosted approach, you have full control over the infrastructure and are responsible for managing the persistence layer, scalability, and availability of the Temporal system.
+
+**Temporal Cloud:**
+Temporal Cloud is a fully managed service provided by Temporal.
+It offers a hassle-free way to run your Temporal Applications without the need to manage the underlying infrastructure.
+Your Workers and Temporal Applications connect to the Temporal Cloud service, which takes care of the persistence layer, scalability, and availability for you.
+
+Temporal Cloud and self-hosted Temporal Services have some similarities, as both require your Temporal Clients and Workers to establish a connection to the Temporal Service.
+Additionally, in both cases, you are responsible for managing and hosting your application code and running your Workers.
+The choice between self-hosted and Temporal Cloud depends on your specific requirements, resources, and expertise.
+
+### Features of Temporal Cloud
+
+Based on the open-source Temporal Server software, Temporal Cloud offers comparable features without the overhead of setting up and managing a production-level Temporal Service.
+It provides [units of isolation called Namespace](/cloud/security#namespace-isolation), which you can provision for different environments such as development, testing, integration, staging, and production.
+
+Temporal Cloud provides additional features and benefits, such as:
+
+- [Globally available](/cloud/service-availability): Temporal Cloud is compatible with applications deployed in various cloud environments or data centers and operates globally.
+- [History Export](/cloud/export): Temporal Cloud allows you to export your Workflow Execution Event Histories for long-term storage, auditing, or analysis purposes.
+- [RBAC](/cloud/saml): Temporal Cloud provides RBAC which supports SAML Single Sign-On (SSO) authentication. This provides a secure and seamless way for users to access Temporal Cloud resources using their organization's credentials, simplifying user management and enhancing security.
+- [Support](/cloud/support): Temporal Cloud includes the support, services and training needed to onboard you successfully, design and deploy your application efficiently and scale. Our team has extensive knowledge of the Temporal project, and a broad set of skills to help you succeed with any project.
+- [Consumption based pricing](/cloud/pricing): Temporal Cloud is a consumption-based service; you pay only for what you need when you need it. Our pricing derives from your use of actions, storage, and support. Pricing is flexible, transparent, and predictable, so you know your costs and never pay for unused capacity.
+
+### High level system topology
+
+There are two major parts of the Temporal Platform that work together to create the fully functioning system.
```

Comment:
This sections seems to be generically describing Temporal, yet it lists Temporal Cloud as the second part. Should this be Temporal Service, in which we state that Temporal Cloud is an offering of the Temporal Service.